README.md

```diff
@@ -8,8 +8,9 @@
 ![](https://github.com/Valerioageno/tuono/actions/workflows/rust.yml/badge.svg)
 ![](https://github.com/Valerioageno/tuono/actions/workflows/typescript.yml/badge.svg)
 
+Adding file to commit history for testing
 
-**Tuono the fastest react framework out there**
+**Tuono is the fastest react framework out there**
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>“React” is a proper noun and needs to be capitalized. (A_GOOGLE[1])
Suggestions: `React`
Rule: https://community.languagetool.org/rule/show/A_GOOGLE?lang=en-US&subId=1
Category: GRAMMAR
<!-- __reviewdog__:ChBiMTA5OWE0ZWVhNWVmYTQ2EgxMYW5ndWFnZVRvb2w= -->